content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -365,3 +366,122 @@ HTTP response code, are used in the HTTP response to the client.
 The currently supported reasons are: `Unauthorized`, `Forbidden`, `Invalid`, `RequestEntityTooLarge`.
 If not set, `StatusReasonInvalid` is used in the response to the client.
 
+### Message Expression
+
+To return a more friendly message when the policy rejects a request, we can use a CEL expression
+to composite a message with `spec.validations[i].messageExpression`. Similar to the validation expression,
```

Comment:
This is useful even before the policy rejects requests because it helps a developer to find and fix mistakes even before any bindings are registered.

Reply:
Is the rejection message returned somewhere other than `ValidatingAdmissionPolicy 'deploy-replica-policy.example.com' with binding 'demo-binding-test.example.com' denied request`? It may be worthwhile to mention it down below too.

Reply:
no worries.